x-pack/plugins/alerting/common/alert.ts

```diff
@@ -61,6 +61,11 @@ export interface AlertAggregations {
   ruleMutedStatus: { muted: number; unmuted: number };
 }
 
+export interface MappedParams extends SavedObjectAttributes {
+  risk_score?: number;
+  severity?: string; // TODO: translate severity to number so it can sortable (20, 40, 60, 80)
```

Comment:
@banderror, To confirm our side conversation. We know that the possibility of severity's values in params are: `critical`, `high`, `medium`, `low`.  We decided that severity in the mapped params, we will be saved as a string in this format: `80-critical`,  `60-high`, `40-medium`, `20-low`

Reply:
Looks good to me 👍